tests/test_spatial_tools.py

```diff
@@ -2,6 +2,7 @@
 
 Author(s):
     Erik S. Holmlund
```

Comment:
@adehecq  and I discussed that we should probably remove the author part of each script, as many others may contribute in the future. For figuring out who did what, there's always `git blame` ;)

Reply:
Removed authors!

Reply:
Or wait, I'll leave that for a separate PR

Reply:
Opened #153